tensorflow/docs_src/get_started/get_started.md

```diff
@@ -29,7 +29,9 @@ working internally when you use the more compact higher level API.
 
 # Tensors
 
-The central unit of data in TensorFlow is the **tensor**. A tensor consists of a
+The central unit of data in TensorFlow is the **tensor**. The formal physicist's and 
+mathematician's definition of a tensor is a multilinear map; the tensors here are in fact the same 
+thing and are simply values in a particular basis. A tensor in Tensorflow consists of a
```

Comment:
Maybe mention numpy.ndarray as very close analogy, as it is familiar to a lot of users:
https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.ndarray.html